recipe/meta.yaml

```diff
@@ -98,15 +92,16 @@ outputs:
       commands:
         - verdi --help
         - verdi --version
-        - pip check
+        # We cannot fix the fixed dependency in aiormq see https://github.com/conda-forge/aiormq-feedstock/issues/59 so we ignore it
+        - "! pip check | grep -v pamqp"
 
   - name: {{ aiidaservice }}
     build:
       noarch: python
     requirements:
       run:
         - postgresql >=9.6
-        - rabbitmq-server >=3.7,<3.8.15
+        - rabbitmq-server >=3.11
```

Comment:
Was this change needed? I think we need to be careful here. If we want to bump rabbitmq, we should try to somehow automatically configure it properly (i.e. override the timeout). Perhaps this should be done as a separate PR?

Reply:
I've reverted this change for now to unblock this PR since I think it requires more discussion. I've created #112 to discuss.